changelogs/fragments/55841-PSLint-errors-fix.yaml

```diff
@@ -0,0 +1,8 @@
+minor_changes:
+        - win_domain - Added a check for pending boot after role installation
+bugfixes:
+        - Fixed PSlint warnings and more
+        - Did not fix - to not break
```

Comment:
I would remove this entry and the below lines, not really useful for end users to know.